ranger/ext/direction.py

```diff
@@ -142,8 +144,16 @@ def move(self, direction, override=None, minimum=0,  # pylint: disable=too-many-
         if self.cycle():
             cycles, pos = divmod(pos, (maximum + offset - minimum))
             self['_move_cycles'] = int(cycles)
-            return int(minimum + pos)
-        return int(max(min(pos, maximum + offset - 1), minimum))
+            ret = minimum + pos
+        else:
+            ret = max(min(pos, maximum + offset - 1), minimum)
+        # Round towards the direction we're moving from.
+        # From the UI point of view, round down. See: #912.
+        if direction < 0:
+            ret = int(math.ceil(ret))
```

Comment:
The documentation makes a point that `math.ceil()` returns a float, which doesn't make sense in our case. It will work but doesn't make sense.